server/openai/test.js

```diff
@@ -3,31 +3,22 @@
  Run via `node openai/test.js` from `server/` or wherever the .env file with the openai key is
 */
 
-const generateItinerary = require('./generateItenerary')
-const Itinerary = require('../models/itineraryModel')
+const generateItinerary = require('./generateTrip')
+const TripModel = require('../models/TripModel')
 
-const exampleItinerary = new Itinerary({
+const exampleItinerary = new TripModel({
   destination: 'Paris',
   budget: 2000,
-  days: 5,
-  stages: 3,
-  preferences: ['shopping', 'museums', 'historical landmarks'],
+  numberOfDays: 5,
+  stagesPerDay: 3,
 })
 
 ;(async function testGenerateItinerary() {
   try {
-    const itineraryConstraints = `
-      Destination: ${exampleItinerary.destination}
-      Budget: $${exampleItinerary.budget}
-      Days: ${exampleItinerary.days}
-      Stages per day: ${exampleItinerary.stages}
-      Preferences: ${exampleItinerary.preferences.join(', ')}
-    `
-
-    const itinerary = await generateItinerary(itineraryConstraints)
+    const itinerary = await generateItinerary(exampleItinerary)
 
     console.log(`Generated itinerary for given constraints:`)
```

Comment:
Remove console logs pls :)

Reply:
this is for the test script. Need it for debugging

Reply:
Sure. Resolving...